packages/adobe-sign/__tests__/services/agreement.service.spec.ts

```diff
@@ -0,0 +1,92 @@
+import "reflect-metadata";
+import { AdobeSignApi } from '../../src/apis/adobe-sign/adobe-sign-api';
+import { Agreement } from '../../src/apis/adobe-sign/models/agreement';
+import { AgreementService } from '../../src/services/agreement.service';
+import * as fixture from "../../__fixtures__/adobe-sign-api";
+
+describe('AgreementService', () => {
+    function setup() {
+        const adobeApi = new AdobeSignApi(null, null);
+        const service = new AgreementService(adobeApi);
+
+        return { service, adobeApi };
+    }
+
+    describe('getAgreement', () => {
+        it('should call the AdobeSignApi with the given id', async () => {
+            // Arrange
+            let { service, adobeApi } = setup();
+            let expectedId = fixture.mockAgreementId;
+            let agreement = new Agreement({
+                id: expectedId,
+                name: '[DEMO USE ONLY] Agreement Test 001',
+                groupId: 'CBJCHBCAABAACnRhBD9pKDAfZ55583n-4WOlwlvI_RGM',
+                type: 'AGREEMENT',
+                status: 'SIGNED',
+                hasFormFieldData: true,
+            });
+
+            let apiSpy = jest.spyOn(adobeApi, 'getAgreement').mockResolvedValue(agreement);
```

Comment:
We have two different patterns in our tests now. I know I started this pattern here, but all of our other spec files now call Mock(TheClass) in `setup` instead of having to call `jest.spyOn` every time.  Could you please clean that piece up?